contrib/win32/openssh/install-sshd.ps1

```diff
@@ -126,3 +127,6 @@ New-Service -Name sshd -DisplayName "OpenSSH SSH Server" -BinaryPathName `"$sshd
 sc.exe privs sshd SeAssignPrimaryTokenPrivilege/SeTcbPrivilege/SeBackupPrivilege/SeRestorePrivilege/SeImpersonatePrivilege
 
 Write-Host -ForegroundColor Green "sshd and ssh-agent services successfully installed"
+
+# add folder to system PATH
+Add-Path -FilePath $scriptdir @psBoundParameters
```

Comment:
Were we going to add a parameter to allow a user to opt-out updating the Machine 'PATH' environment variable?

Reply:
For now, running .\install-sshd.ps1 -confirm will provided interactive way to opt-out